src/functions/GetLocation.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-console */
+const setGeoOnSuccess = (position) => {
+  console.log(position.coords.latitude);
+  console.log(position.coords.longitude);
+};
+
+const setGeoOnError = () => {
+  return "Not available";
+};
+
+const getLocation = async () => {
+  const position = await new Promise((resolve, reject) => {
+    navigator.geolocation.getCurrentPosition(resolve, reject);
+  }).catch(setGeoOnError);
+
+  return position.coords ? position.coords : position;
```

Comment:
Not a big deal: You can use new [Nullish coalescing operator here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)